packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -269,6 +269,58 @@ typedef RefreshCallback = Future<void> Function();
 /// sliver such as [CupertinoSliverNavigationBar] and your main scrollable
 /// content's sliver.
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// When the user scrolls past [refreshTriggerPullDistance],
+/// this sample shows the default ios pull to refresh indicator for 1 second and
+/// adds a new item to the top of the list view.
+///
+/// ```dart imports
+/// import 'package:flutter/cupertino.dart';
+/// ```
+///
+/// ```dart
+/// List<Color> colors = [Colors.amber[100], Colors.amber[300], Colors.amber[600]];
+/// List<Widget> items = [
+///   Container(color: Colors.amber[600], height: 100.0),
+///   Container(color: Colors.amber[300], height: 100.0),
+///   Container(color: Colors.amber[100], height: 100.0),
+/// ];
+///
+/// Future<void> _getItems() async {
+///   await Future.delayed(Duration(milliseconds: 1000));
+///   setState(() {
+///     items.insert(0, Container(color: colors[items.length % 3], height: 100.0));
+///   });
+/// }
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   return MaterialApp(
```

Comment:
Since it's a cupertino example, I'd just use CupertinoApp and CupertinoPageScaffold here. (I also suspect this doesn't build if you just import cupertino.dart above)

Reply:
> (I also suspect this doesn't build if you just import cupertino.dart above)

I referred to this example.
https://api.flutter.dev/flutter/cupertino/CupertinoContextMenu-class.html